tasks/install.yml

```
@@ -44,6 +44,7 @@
   register: tailscale_status
   failed_when:
     - tailscale_status.rc != 0
+    - tailscale_status.rc != 1
     - "'Logged out.' not in tailscale_status.stdout"
```

Alternative to retain the granularity of the current check:
```suggestion
    - "'Logged out.' not in tailscale_status.stdout or 'not logged in' not in tailscale_status.stdout"
```